submissions/DmytryjK/memory-pair-game/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Memory pair game</title>
+    <link href="https://fonts.cdnfonts.com/css/gagalin" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <main class="main">
+        <ul class="main__cards"></ul>
+        <div class="winMessage">
+            <h2 class="winMessage__title">You're a winner!</h2>
+            <img class="winMessage__image" src="images/winMessage.png" alt="">
+            <button class="click-to-continue">click to continue</button>
+        </div>
+    </main>
+    <section class="timer">
```

Comment:
On some resolutions, you have a bug with container width: [video](https://share.vidyard.com/watch/93BjC3vaKWEtzDucubBVBy?)

Reply:
fixed